src/components/common/MobileMenu.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <menu>
+    <router-link
+      class="mobile-menu-item"
+      to="/portfolio"
```

Comment:
should we extract the links from the desktop and mobile menu?

Reply:
what do you mean?

Reply:
make the links a list that the components receive so we don't have to maintain the links in two places

Reply:
nice refactor idea

Reply:
Good idea. Same for the bottom bar.